sources/dialect/README.MD

```diff
@@ -0,0 +1,19 @@
+# Dialect Support
+
+This rule has net yet been added.
+For now, we have only added some files and sources.
+This section will gradually be expanded as the rule is written.
+
+The files inside ``original_files`` were added from the version 1 tree of the original repository, [``en-wl/wordlist``](https://github.com/en-wl/wordlist/tree/v1).
+
+## Sources
+
+1. VarCon Documentation
+    - Link: [Variant Conversion Info (VarCon)](https://github.com/en-wl/wordlist/blob/v1/varcon/README)
+    - License: [MIT-like](https://raw.githubusercontent.com/kevina/wordlist/master/scowl/Copyright)
```

Comment:
You should mention the fact the license is mentioned in the README.md file and provide the link to the README, 

Then mention you provide here the link to copyright URL mentioned in the README, because otherwise, people would be surprised to see a link that mean nothing.

It looks like the var link was updated but the link wasn't

Reply:
Maybe using a Markdown ref with footer reference would help to provide context and would avoid to duplicate the explanations

Reply:
> You should mention the fact the license is mentioned in the README.md file and provide the link to the README,

I don't believe this is necessary. People are somewhat familiar with the phrase `MIT-like` and adding a detailed explanation for how I used that phrase seems redundant. 

> Then mention you provide here the link to copyright URL mentioned in the README, because otherwise, people would be surprised to see a link that mean nothing.
>
> It looks like the var link was updated but the link wasn't

Sorry, I'm not sure what you meant here?

Reply:
The original is:
```
1. VarCon Documentation
    - Link: [Variant Conversion Info (VarCon)](https://github.com/en-wl/wordlist/blob/v1/varcon/README)
    - License: [MIT-like](https://raw.githubusercontent.com/kevina/wordlist/master/scowl/Copyright)
    - Retrieved at: 2024-08-03
```

Do you mean to say you would prefer something like:
```
1. VarCon Documentation
    - Link: [Variant Conversion Info (VarCon)](https://github.com/en-wl/wordlist/blob/v1/varcon/README)
    - License: [MIT-like](https://raw.githubusercontent.com/kevina/wordlist/master/scowl/Copyright)(described as such by the author in the titular [README.md](https://github.com/en-wl/wordlist/blob/v1/README.md))
    - Retrieved at: 2024-08-03
```
(You can compare these in a Markdown preview)

Reply:
Yes, it's what I suggested